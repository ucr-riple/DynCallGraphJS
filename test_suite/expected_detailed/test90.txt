{
    'global (test90.js:1:1:7:5)': [
        'anon (test90.js:1:2:7:2)'
    ],
    'anon (test90.js:5:15:5:20)': [
        'foo (test90.js:2:3:4:4)'
    ],
    'anon (test90.js:6:3:6:10)': [
        'bar (test90.js:3:12:3:29)'
    ]
}