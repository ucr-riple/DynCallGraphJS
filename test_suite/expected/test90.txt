{ '(test90.js:1:1:7:5)':
   [ '(test90.js:1:2:7:2)' ],
  '(test90.js:5:15:5:20)':
   [ '(test90.js:2:3:4:4)' ],
  '(test90.js:6:3:6:10)':
   [ '(test90.js:3:12:3:29)' ] }